Resources/Private/JavaScript/Host/Containers/LeftSideBar/index.js

```diff
@@ -10,17 +10,20 @@ import style from './style.css';
 const {$get} = immutableOperations;
 
 @connect(state => ({
-    isHidden: $get(state, 'ui.leftSideBar.isHidden')
+    isHidden: $get(state, 'ui.leftSideBar.isHidden'),
+    isFullScreen: $get(state, 'ui.fullScreen.isFullScreen')
 }))
 export default class LeftSideBar extends Component {
     static propTypes = {
-        isHidden: PropTypes.bool.isRequired
+        isHidden: PropTypes.bool.isRequired,
+        isFullScreen: PropTypes.bool.isRequired
     };
 
     render() {
         const classNames = mergeClassNames({
             [style.leftSideBar]: true,
-            [style['leftSideBar--isHidden']]: this.props.isHidden
+            [style['leftSideBar--isHidden']]: this.props.isHidden,
```

Comment:
The `isFullScreen` class could be removed, not only would the UX benefit from this, since the transform is animated, the browser could also improve the performance of the rendering, since mutating the `display`property would cause a reflow, and `transform` only a repaint.

I would just merge both props into one, f.e.
```
[style['leftSideBar--isHidden']]: this.props.isHidden || this.props.isFullScreen
```